src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/TvzDomainSections.java

```diff
@@ -1,41 +1,53 @@
 package nz.cri.gns.NZSHM22.opensha.inversion;
 
+import com.google.common.base.Preconditions;
+import nz.cri.gns.NZSHM22.opensha.enumTreeBranches.NZSHM22_FaultModels;
+import nz.cri.gns.NZSHM22.opensha.enumTreeBranches.NZSHM22_LogicTreeBranch;
+import nz.cri.gns.NZSHM22.opensha.faults.FaultSectionList;
+import nz.cri.gns.NZSHM22.opensha.faults.NZFaultSection;
 import org.opensha.commons.data.CSVFile;
-import org.opensha.commons.geo.GriddedRegion;
 import org.opensha.commons.util.modules.helpers.CSV_BackedModule;
 import org.opensha.sha.earthquake.faultSysSolution.FaultSystemRupSet;
-import org.opensha.sha.earthquake.faultSysSolution.modules.PolygonFaultGridAssociations;
 import org.opensha.sha.faultSurface.FaultSection;
 
-import java.awt.geom.Area;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
-import java.util.function.IntPredicate;
+import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
-public abstract class RegionSections implements CSV_BackedModule {
+
+public class TvzDomainSections implements CSV_BackedModule {
 
     protected Set<Integer> sections;
 
-    public RegionSections() {
+    public TvzDomainSections() {
```

Comment:
is it appropriate to name this specifically for TVZ or should it (and other fns and classes) have generic names?